charts/consul/test/unit/server-acl-init-job.bats

```diff
@@ -1483,64 +1610,36 @@ load _helpers
 #--------------------------------------------------------------------
 # global.acls.bootstrapToken
 
-@test "serverACLInit/Job: -bootstrap-token-file is not set by default" {
+@test "serverACLInit/Job: bootstrapToken.secretKey is required when bootstrapToken.secretName is set" {
```

Comment:
these tests have changed because we now fail if only one of secretKey or secretName has been provided